.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
should this be a .gitkeep file instead?

Reply:
apparently not (this is the automated install from husky)